girder-dandi-archive/girder_dandi_archive/rest.py

```diff
@@ -69,21 +73,84 @@ def create_dandiset(self, params):
     @describeRoute(
         Description("Get Dandiset").param("identifier", "Dandiset Identifier", paramType="path")
     )
+    @dandiset_identifier
     def get_dandiset(self, identifier, params):
-
-        if not identifier:
-            raise RestException("identifier must not be empty.")
-
-        if not validate_dandiset_identifier(identifier):
-            raise RestException("Invalid Dandiset Identifier")
-
         # Ensure we are only looking for drafts collection child folders.
         drafts = get_or_create_drafts_collection()
         doc = Folder().findOne({"parentId": drafts["_id"], "meta.dandiset.identifier": identifier})
         if not doc:
             raise RestException("No such dandiset found.")
         return doc
 
+    @access.public
+    @describeRoute(
+        Description("Get Dandiset Owners").param(
+            "identifier", "Dandiset Identifier", paramType="path"
+        )
+    )
+    @dandiset_identifier
+    def get_dandiset_owners(self, identifier, params):
+        return Folder().getFullAccessList(find_dandiset_by_identifier(identifier))["users"]
```

Comment:
you should only return those with `ADMIN` access

Reply:
Good catch, addressed in 753cce0